### DISCUSSION TITLE
hello everyone I m now setting up Nocodb [May 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2109] [Discord Threads]
> 🙋‍♂️ **wwaaauuuu**

**Comment by o1lab** [comment]
> 🙋‍♂️ **CaptainKranch**  
#### Hello backend isn't my field either but when I deleted the docker container and docker image to update to the last one, it saved the previous information. I'm using MySQL btw. But, I really suggest you to make a backup if it is important information!  
#  
> 🙋‍♂️ **CaptainKranch**  
#### Localhost as you mentioned is your computer acting as a server. The IP would be 127.0.0.1 and the port would be 8080 sooo if you want yo share your project you can open a port for another person to use it with SSH and just open the localhost but that kinda complicated if you do not know how  
#  
> 🙋‍♂️ **CaptainKranch**  
#### so you can use the heroku option, its free if you do not have a big project and anyone would be able to access your project if you invite him  
#  
> 🙋‍♂️ **wwaaauuuu**  
#### oh, it sounds great. I'll try to upgrade then as said in manual.  
#  
> 🙋‍♂️ **wwaaauuuu**  
#### don't you know if my guest should be on the same network? or ssh tunnel doesn't require it? don't you know any software with gui which makes the ssh tunneling easier?  
#  
> 👨‍🚀 **wingkwong**  
#### For Q1 - It depends. If you haven't defined NC_DB when starting docker, then a sqlite file as a meta database will be created under /usr/app/data. If you create a project without connecting to external database, then data will be stored in this sqlite db as well. In this case, deleting the docker container will erase that as well. One way to persist that is to mount volume, which we've included in our starter guideline - See https://github.com/nocodb/nocodb#docker.     
####    
#### Therefore, you need to backup if & only if   
####    
#### - you didn't mentioned NC_DB when starting our app and   
#### - you didn't mount your local volume at /usr/app/data/  
#  
> 👨‍🚀 **wingkwong**  
#### For Q2 - that is not accurate. `localhost:8080` is just a sample only. You can deploy NocoDB to different places and it would serve as <HOST>:<PORT>. We use localhost just for people to setup locally to give it a spin. Like captainKranch said, sharing locally needs some setup. If you wanna collaborate with someone, you may try Heroku deployment. That would be the easiest way for you.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972214791873646602)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
